File: pytools/data/data.code.php.server/ReqChatGm.php
<?php
namespace protocol;

use network\Packet;


class ReqChatGm
{
	private $content = '';


	public function __construct(&$packet)
	{
		$this->decode($packet);
	}

	public function decode(&$packet)
	{
		$this->content = $packet->readString();
	}


	public function getContent()
	{
		return $this->content;
	}

}

File: pytools/data/data.code.java.server/ReqChatGm.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class ReqChatGm
{
	private String content;


	public ReqChatGm(Packet packet)
	{
		content = packet.readString();
	}



	public String getContent()
	{
		return this.content;
	}

}
